package.json

```diff
@@ -11,6 +11,7 @@
     "lodash": "^4.17.10",
     "moment": "^2.22.2",
     "pako": "^1.0.6",
```

Comment:
we should be able to remove `pako` now right?

Reply:
We have still some problems here. Probably i'm missing something with Protobuf. 
I'm checking this together with @ignacionr